osfoffline/polling_osf_manager/remote_objects.py

```diff
@@ -73,7 +73,7 @@ def __init__(self, remote_dict):
     def validate(self):
         super().validate()
         assert self.provider
-        assert self.move_url if not self.id  else True
+        assert self.move_url if not self.id else True
```

Comment:
Am reviewing this a bit backwards to divide-and-conquer.

Done reviewing all items below this line.

(Arguably we're misusing asserts here in potentially freeze-inducing ways, but I won't hold that here since this is only cleanup to existing code)